README.md

```diff
@@ -69,9 +70,9 @@
 
 > Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[Adding a Gym service]**
+- **[Register and login using a username]**
+- **[Reserving a gym service]**
 
```

Comment:
You ought to make this README more specific to the applkication.

I am also not seeing any documentation for the API it self. Please consider adding this (You can make your own or yo ucan use something like swagger. See the [rswag](https://github.com/rswag/rswag) gem for more info)

Otherwise, good job!

Additionally, you ought to consider making the instructions to install clearer - Make it more specific to this application

Reply:
[OPTIONAL] 
Consider wrapping your install commands with backtick  rags like so `rails db:create` which would look like